dash-pipeline/bmv2/stages/outbound_routing.p4

```diff
@@ -33,8 +50,13 @@ control outbound_routing_stage(inout headers_t hdr,
             return;
         }
 
+        routing_group.apply();
+        if (meta.eni_data.routing_group_admin_state) {
         routing.apply();
+        } else {
```

Comment:
nit: drop and return first might be better then if else : D

Reply:
Done